scripts/run-image.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+# why this script you may ask?
+# need to be able to clean up the docker image from a crtl-c
+# and it's not straightforward in a Makefile.
+
+# trap ctrl-c and call ctrl_c()
+trap ctrl_c INT
+
+function ctrl_c() {
+    docker rm -f ${CONTAINER_ID}
+    exit 0
+}
+
+CONTAINER_ID=$(docker run -d -e ACCESS_ID=1234 -e ACCESS_SECRET=1234 -e PNBJ_LOGLEVEL=debug -p 9090:9090 -p 50051:50051 pbnj:local)
```

Comment:
Maybe we should expose :8080 which is the metrics and http healthcheck endpoint as well?

Reply:
yeah, good point. will do

Reply:
updated